recipes/suffix_tree/LICENSE.txt

```diff
@@ -0,0 +1 @@
+UNKNOWN
```

Comment:
This package cannot be merged unless we have clear rights to distribute it. Sorry!